nixpkgs/nixpkgs.bzl

```diff
@@ -15,6 +15,28 @@ def python(repository_ctx):
     if python_path:
         return python_path
 
+def _implode_packages_defs(def_files, attribute_paths):
+    """Merge the two dics passed as arguments into a dict of dicts"""
```

Comment:
This is a bit more specific than the docstring makes it out to be, it looks like.